scrapy/core/scheduler.py

```diff
@@ -52,12 +54,13 @@ def from_crawler(cls, crawler):
         dupefilter_cls = load_object(settings['DUPEFILTER_CLASS'])
         dupefilter = create_instance(dupefilter_cls, settings, crawler)
         pqclass = load_object(settings['SCHEDULER_PRIORITY_QUEUE'])
+        dpqclass = load_object(settings['SCHEDULER_DELAYED_REQUESTS_PRIORITY_QUEUE'])
```

Comment:
What about shortening it to `SCHEDULER_DELAY_QUEUE` (or `SCHEDULER_TIME_QUEUE`)?